samples/ProjectGrandmaster.Unity/Assets/Script/Board/BoardRotateHandle.cs

```diff
@@ -0,0 +1,95 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+// Contributed by USYD Team - Hrithvik (Jacob) Sood, John Tran, Tom Derrick, Aydin Ucan, Aayush Jindal
+
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using System;
+using Microsoft.MixedReality.Toolkit.UI;
+using Microsoft.MixedReality.SpectatorView.ProjectGrandmaster;
+
+namespace Microsoft.MixedReality.SpectatorView.ProjectGrandmaster
+{
+    public class BoardRotateHandle : MonoBehaviour
+    {
+        public GameObject chessBoard;
+
+        GameObject gameManager;
+        BoardInformation boardInfo;
+        PieceAction pieceAction;
+
+        PieceInformation.Colour lossColour;
+
+        void tiltForfeit(PieceInformation.Colour colour)
```

Comment:
Methods should be PascalCase, ensure for the rest below.

Also add the explicit modifier here, for methods default is internal, but I presume this is meant to be private?